### ISSUE TITLE
rt1 - Text.empty yields unison: user error (type error, expecting T)

### ISSUE THREAD

**Issue by francisdb** [unisonweb/unison#351]
```
> "" == "test"
```
works as expected

```
  ┌
  │  2 | > "" == "test"
  │        ⧩
  │        false
  └
```

but
```
> Text.empty == "test"
```

runtime crash with
```
unison: user error (type error, expecting T)
```

**Comment by aryairani** [comment]
```
let
  arg0 = (Lambda 0 "")
  arg1 = "test"
  ((Lambda 2 External ##Text.==) @1 @0)
```
`Text.empty` is apparently an unapplied lambda of 0 args instead of a value.

**Comment by aryairani** [comment]
Fixed in 26666d4b5380df7ada038734e9ebee649b4a487d.
